mappings/com/mojang/blaze3d/platform/GlStateManager.mapping

```diff
@@ -348,6 +351,9 @@ CLASS net/minecraft/class_1492 com/mojang/blaze3d/platform/GlStateManager
 		FIELD field_29094 capFill Lnet/minecraft/class_1492$class_5849;
 		FIELD field_29096 factor F
 		FIELD field_29097 units F
+	CLASS class_5863 RenderPhaseTexturing
```

Comment:
hm?

Reply:
check out RenderPhase.Texturing in newer versions
serves the same purpose

Reply:
How about `RenderMode`? Another 1.14 name

Reply:
that’s an enum  that sets and unsets  texturing methods too? then sure

Reply:
There are literally the exact same enums.

Reply:
kek

Reply:
ok

Reply:
dont resolve until he makes the changes